### PR TITLE
gd 2.1.1

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/gd.rb [hunk 1]
```diff
@@ -32,7 +35,9 @@ class Gd < Formula
 
   def install
     ENV.universal_binary if build.universal?
-    args = %W{--disable-dependency-tracking --prefix=#{prefix}}
+
+    args = "--disable-dependency-tracking",
+           "--prefix=#{prefix}"
```
Comment (xu-cheng) [inline Library/Formula/gd.rb]: We prefer:
```ruby
args = %W[
  --disable-dependency-tracking
  --prefix=#{prefix}
]
```

Thanks.

File: Library/Formula/gd.rb [hunk 2]
```diff
@@ -32,8 +35,11 @@ class Gd < Formula
 
   def install
     ENV.universal_binary if build.universal?
-    args = %W{--disable-dependency-tracking --prefix=#{prefix}}
 
+    args = %W[
+      "--disable-dependency-tracking"
+      "--prefix=#{prefix}"
+    ]
```
Comment (jacknagel) [inline Library/Formula/gd.rb]: The quotes aren't necessary (and will break) when using a `%W[]` array.

Comment (dunn) [inline Library/Formula/gd.rb]: Ohhh, right. Sorry about that.

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**